libmscore/rendermidi.h

```diff
@@ -0,0 +1,116 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2019 Werner Schweer and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program; if not, write to the Free Software
+//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+//=============================================================================
+
+#ifndef __RENDERMIDI_H__
+#define __RENDERMIDI_H__
+
+#include "fraction.h"
+#include "measure.h"
+
+namespace Ms {
+
+class EventMap;
+class MasterScore;
+class Staff;
+class SynthesizerState;
+
+enum class DynamicsRenderMethod : signed char {
```

Comment:
+1 for moving all this into a header. I was thinking about doing the same when I had to add this enum and other stuff...